syntax = "proto3";
package prompt_builder;

service PromptBuilder {
  rpc BuildPrompt (BuildPromptRequest) returns (BuildPromptResponse);
}

// Request parameters for BuildPrompt
message BuildPromptRequest {
    string req_id = 1;
    string client_id = 2;
    int32 prompt_id = 3;
    string source_id = 4;
    string from_time = 5;
    string to_time = 6;
}

// Response for BuildPrompt
message BuildPromptResponse {
    string prompt = 1;
}